medusa/providers/torrent/rss/nyaa.py

```diff
@@ -143,5 +143,21 @@ def parse(self, data, mode):
 
         return items
 
+    def _get_season_search_strings(self, episode):
+        search_string = {
+            'Season': []
+        }
+        season_search_keywords = ('batch', 'complete', 'season')
```

Comment:
Currently non anime doesn't need that as we don't support downloading `complete` releases, which are the complete show. Scene anime also follows normal "show's" numbering.

This could be usefull for foreign or non "standard" naming, like for example instead of `Season 1` releases with `Series 1` that exist.
So something like yoou propose would be usefull still for everything.